composer/algorithms/seq_length_warmup/seq_length_warmup.py

```diff
@@ -197,7 +197,7 @@ def apply(self, event: Event, state: State, logger: Logger) -> Optional[int]:
                 raise RuntimeError("'labels' must be in model inputs")
 
             # create fake inputs
-            vocab_size = len(self._original_model.tokenizer)
+            vocab_size = self._original_model.config.vocab_size
```

Comment:
Why is this necessary? We are creating dummy inputs to do 1 fwd/bkw, so can we fill it with all zeros? That way we don't have to rely on the model having `.config` attribute.